_board/diodes_delight_piunora.md

```diff
@@ -8,6 +8,7 @@ board_url: "https://www.crowdsupply.com/diodes-delight/piunora"
 board_image: "diodes_delight_piunora.jpg"
 download_instructions: 
 date_added: 2021-2-14
+family: raspberrypi
```

Comment:
Want to call this broadcom to distinguish from rp2040? Thats what we do it CP.

Reply:
Sure